libkbfs/data_types.go

```diff
@@ -58,6 +58,7 @@ type TeamInfo struct {
 	TID          keybase1.TeamID
 	CryptKeys    map[KeyGen]kbfscrypto.TLFCryptKey
 	LatestKeyGen KeyGen
+	RootID       keybase1.TeamID // for subteams only
```

Comment:
Are we using `RootID` being non-empty as an indication for a team is a sub-team?